ui/raidboss/data/05-shb/alliance/the_copied_factory.txt

```diff
@@ -57,7 +57,7 @@ hideall "--sync--"
 ### Trash
 # -ic 2P -ii 491D 491C -p 491B:715.5
 600.0 "--sync--" sync /00:0839:Warehouse B will be sealed off/ window 600,0
-700.0 "--sync--" sync / 22:........:Small Flyer:........:Small Flyer:01/ window 100,0
+700.0 "--sync--" sync /22:........:Small Flyer:........:Small Flyer:01/ window 100,0
```

Comment:
This is not necessary, but seems fine.

Reply:
yeah i know but i like to have things consistent :)